src/contracts/interfaces/IRegistryCoordinator.sol

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.12;
+
+/**
+ * @title Interface for a contract the coordinates between various registries for an AVS.
```

Comment:
typo: 'the' => 'that'

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/48/commits/2580b31c45e16badb4b3870a6ee7b25c93514470